[urdfdom] Fixing up _IMPORT_PREFIX

seanyen opened the pull request:
In the resulting `urdfdom-config.cmake`, it contains an uninitialized variables `${_IMPORT_PREFIX}`. The patch is fixing it up by replacing with `${CMAKE_CURRENT_LIST_DIR}/../..`.

seanyen committed:
Fixing up _IMPORT_PREFIX.

seanyen committed:
update version.

JackBoosY commented on ports/urdfdom/portfile.cmake:
```diff
@@ -29,6 +29,10 @@ else()
 endif()
 
+file(READ ${CURRENT_PACKAGES_DIR}/share/urdfdom/urdfdom-config.cmake _contents)
+string(REPLACE "\${_IMPORT_PREFIX}" "\${CMAKE_CURRENT_LIST_DIR}/../.." _contents "${_contents}")
```
Should patch `urdfdom-config.cmake` and add the following codes instead of this:
```
get_filename_component(_IMPORT_PREFIX "${CMAKE_CURRENT_LIST_FILE}" PATH)
get_filename_component(_IMPORT_PREFIX "${_IMPORT_PREFIX}" PATH)
```

traversaro commented:
Hi @seanyen, thanks for working on this! This PR is currently blocking https://github.com/microsoft/vcpkg/pull/13320 . Do you plan to work on it in the future? Otherwise I can try to complete it myself, thanks!

seanyen committed:
address the feedback.

seanyen commented:
@JackBoosY  I have updated this pull request. Let me know if any changes required!

@traversaro Hi! Thanks for the ping. I just updated this pull request. Hope this one can go through soon.

ras0219-msft commented:
Thanks for the PR!
